charts/nango/Chart.lock

```diff
@@ -2,8 +2,8 @@ dependencies:
 - name: postgresql
   repository: https://charts.bitnami.com/bitnami
   version: 13.2.24
-- name: temporal
-  repository: https://airbytehq.github.io/helm-charts
-  version: 0.50.20
-digest: sha256:8c2cd0f4d2fe1c08608d40f83b07efb726997ef1126737096f5f49b7d7be254f
-generated: "2023-12-20T17:06:21.930252-05:00"
+- name: elasticsearch
+  repository: https://charts.bitnami.com/bitnami
+  version: 21.3.3
+digest: sha256:df557f6fd1ac5231b28abb6a176c5b599e66d038bb755ac306e4464b611680c9
+generated: "2024-07-23T15:36:44.302022-04:00"
```

Comment:
Are we sure we want that? 
I'm not sure how stable that is. 
Also where does the version comes from? When I look at binami it's not this https://bitnami.com/stack/elasticsearch/helm

Reply:
Comes from

```
➜  helm-charts git:(khaliq/nan-1472-update-helm-charts-temporal-removal) helm search repo bitnami/elasticsearch --versions
NAME                 	CHART VERSION	APP VERSION	DESCRIPTION                                       
bitnami/elasticsearch	21.3.3       	8.14.3     	Elasticsearch is a distributed search and analy...
bitnami/elasticsearch	21.3.2       	8.14.3     	Elasticsearch is a distributed search and analy...
bitnami/elasticsearch	21.3.1       	8.14.3     	Elasticsearch is a distributed search and analy...
bitnami/elasticsearch	21.3.0       	8.14.2     	Elasticsearch is a distributed search and analy...
```

Reply:
We're running 8.14.3 which is what I based that version on